examples/table/mui/src/pages/table/list.tsx

```diff
@@ -0,0 +1,527 @@
+import { alpha } from "@mui/material/styles";
+import {
+    DeleteButton,
+    EditButton,
+    SaveButton,
+    Button,
+    TextField,
+    Stack,
+    Toolbar,
+    Typography,
+    MenuItem,
+    Tooltip,
+    Box,
+    TableContainer,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    Checkbox,
+    Paper,
+    TableBody,
+    TableSortLabel,
+    TablePagination,
+} from "@pankod/refine-mui";
+
+import { useForm, Controller } from "@pankod/refine-react-hook-form";
+import "react-mde/lib/styles/css/react-mde-all.css";
+
+import {
+    useTable,
+    Column,
+    usePagination,
+    useExpanded,
+    useRowSelect,
+    useSortBy,
+    useFilters,
+} from "@pankod/refine-react-table";
+import ReactMarkdown from "react-markdown";
+import ReactMde from "react-mde";
+
+import { IPost, ICategory } from "interfaces";
+
+export const PostList: React.FC = () => {
+    const [selectedTab, setSelectedTab] =
+        useState<"write" | "preview">("write");
+
+    const {
+        refineCore: { onFinish, id, setId },
+        register,
+        handleSubmit,
+        control,
+    } = useForm<IPost>({
+        refineCoreProps: {
+            resource: "posts",
+            redirect: false,
+            action: "edit",
+        },
+    });
+
+    const { mutate } = useDeleteMany<IPost>();
+
+    const deleteSelectedItems = (ids: string[]) => {
+        mutate({
+            resource: "posts",
+            ids,
+        });
+    };
+
+    const columns: Array<Column> = React.useMemo(
+        () => [
+            {
+                id: "id",
+                Header: "ID",
+                accessor: "id",
+            },
+            {
+                id: "title",
+                Header: "Title",
+                accessor: "title",
+                filter: "contains",
+            },
+            {
+                id: "category.id",
+                Header: "Category",
+                accessor: "category.id",
+                Cell: ({ value }) => {
+                    const { data } = useOne<ICategory>({
+                        resource: "categories",
+                        id: value,
+                    });
+                    return data?.data.title || "loading";
+                },
+                filter: "eq",
+            },
+            {
+                id: "actions",
+                Header: "Actions",
+                accessor: "id",
+                //eslint-disable-next-line react/display-name
+                Cell: ({ value }: any) => {
+                    return (
+                        <Stack direction="row">
+                            <EditButton
+                                onClick={() => {
+                                    handleEditButtonClick(value);
+                                }}
+                            >
+                                Edit
+                            </EditButton>
+                            <DeleteButton
```

Comment:
```suggestion
                            <DeleteButton
```
we don't need to override onClick